docs/source/setup-initial-environment.md

```diff
@@ -12,11 +12,17 @@ If you want to use this template for your AI project, go to the project template
 
 To begin, you'll need to fork this repository to create your own copy. If you're unsure how, look at [Fork a Repo](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) from GitHub docs.
 
-## 2. Access JupyterHub and spawn the Elyra image
+## 2. Access the Elyra image in JupyterHub
 
-### Operate First access JupyterHub
+### Access JupyterHub set up by Project Meteor
 
-1. You can get access to [JupyterHub][3] on [Operate First][2] using the following [link](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/).
+If you are viewing this tutorial in the Jupyter Book rendered by [Project Meteor][2], you can simply hover over the rocket icon on the top right and click JupyterHub. This will take you to a JupyterLab environment spawned with the Elyra AIDevSecOps image.
```

Comment:
You also gain access to Jupyterlab directly from the spawning page on the Meteor web app. Not sure if you want to mention that route as well?

Reply:
Yeah sure, I can mention that as well!